styles/contact.css

```diff
@@ -22,3 +22,43 @@
     -webkit-background-clip: text;
     -webkit-text-fill-color: transparent;
 }
+
+@media (max-width: 863px) {
+    .contact-information .text-contact h1,
```

Comment:
É melhor não adicionar estilos no elemento `H1` diretamente, nesse caso é melhor criar uma classe pra esse titulo
```suggestion
    .contact-information .text-contact .contact-information-title,
```

Reply:
![image](https://github.com/mikaelelias/portfolio/assets/129218135/d80be377-6109-4146-8cb4-9477ee163f84)

Fica com o visual meio estranho, mas como pode até 3 class, de boa né?